wikipedia/track.py

```diff
@@ -130,14 +140,14 @@ def partition(self, partition_index, total_partitions):
         return self
 
     def params(self):
+        ids = ids_samples()
         rules = []
         for i in range(self.query_ruleset_params.ruleset_size):
-            # Note: `AccessibleComputing` is one of the _ids indexed in the dataset
             rule = {
                 "rule_id": "rule_{{i}}",
-                "type": "pinned",
+                "type": random.choice(["pinned", "exclude"]),
```

Comment:
Learning purposes: We are choosing either one in this scenario. Do we need rule type where both of these presents similar to your API tests (If I remember correctly)? Or it is avoided because having both does not make any impact on the performance?

Reply:
The idea is that there will likely be multiple matching rules, so this is just trying to get a read on performance.